src/processor/operator/recursive_extend/path_property_probe.cpp

```diff
@@ -65,9 +65,24 @@ static void copyInternalID(ValueVector* srcVector, ValueVector* dstIDVector,
     }
 }
 
+// TODO(Xiyang): revisit me. Instead of printing in src->dst order. Maybe left->right order make
```

Comment:
I would do this todo now. It will be much more difficult to come back to this to handle these edge cases.

Reply:
Yeah I'll do this before the release.